templates/docs/examples/patterns/grid/grid-breakout-no-aside.html

```diff
@@ -0,0 +1,20 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Grid / Wide - no aside{% endblock %}
+
+{% block standalone_css %}patterns_grid-breakout{% endblock %}
+
+{% block content %}
+
+<style>
+  .viz-grid-item, .viz-aside {
```

Comment:
This seems to be repeated a lot in these examples. Maybe extract it to separate file and `{% include %}` here?

Reply:
done